Lib/glyphsLib/interpolation.py

```diff
@@ -177,13 +179,16 @@ def write_axes(axes, writer):
             axisElement.append(labelname)
 
 
-def find_regular_master(masters):
+def find_regular_master(masters, regularName=None):
     """Find the "regular" master among the master UFOs.
     """
     assert len(masters) > 0
     base_family = masters[0].info.familyName
     assert all(m.info.familyName == base_family for m in masters), \
         'Masters must all have same family'
+    for font in masters:
+        if font.info.styleName == regularName:
```

Comment:
would this mean that if a `font.info.styleName` happens to be None, thus it is equal to the default `regularName=None`, it would be chosen as the regular master?